sql/2020/22_HTTP_2/adoption_of_http_2_by_site_and_requests.sql

```diff
@@ -0,0 +1,20 @@
+#standardSQL
+# 22_HTTP_2 - Adoption of HTTP/2 by site and requests
```

Comment:
FWIW here and throughout it's ok to drop the `22_HTTP_2` prefix.

Reply:
Sounds good. Dropping the `22_HTTP_2` prefix in all files